### PR TITLE
Octopus support

### PR DESCRIPTION
**Pull request by Tpelaaja**
This adds octopus support for NBMiner & T-Rex.

### CODE REVIEW

File: internal/miners.json [hunk 1]
```diff
@@ -129,7 +131,8 @@
                 "darwin": "https://github.com/ethereum-mining/ethminer/releases/download/v0.18.0/ethminer-0.18.0-cuda-9-darwin-x86_64.tar.gz"
             },
             "algos": [
-                "ethash"
+                "ethash",
+                "etchash"
```
Comment (TheBlueBurger) [inline internal/miners.json:135]: This PR is unrelated to etchash?

File: internal/pools.json [hunk 2]
```diff
@@ -20,8 +20,13 @@
             },
             "autolykos2": {
                 "host": "stratum+tcp://autolykos.REGION.nicehash.com:3390"
+            },            
+            "octopus": {
+                "host": "stratum+tcp://octopus.REGION.nicehash.com:3389"
             }
 
+            
+
```
Comment (TheBlueBurger) [inline internal/pools.json:29]: Why did you add 2 newlines?

**Review by TheBlueBurger — CHANGES_REQUESTED** [review]

File: internal/miners.json (hunk 1 above)
Comment (Tpelaaja) [reply internal/miners.json:135]: Yeah it might be probably from older pr.

Ill try to make some shit again as that's from an older pr

**Comment by Tpelaaja** [comment]
Due to a problem with my main branch its fucked

**Comment by Vukkyy** [comment]
> Due to a problem with my main branch its fucked

never, ever ever EVER, make PRs from the main branch
make separate branches for the PRs you make, like `add-octopus`
make your main branch the "Master Code"

**Comment by Tpelaaja** [comment]
Kk
